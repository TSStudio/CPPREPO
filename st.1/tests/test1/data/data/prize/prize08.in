5
tony 88 88 Y
roy 99 99 N
resy 78 78 N
hanke 88 78 N
miae 78 87 Y