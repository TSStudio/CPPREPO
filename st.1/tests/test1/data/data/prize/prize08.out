roy
12000
24000